tabs/pid_tuning.js

```diff
@@ -701,11 +718,8 @@ TABS.pid_tuning.initRatesPreview = function () {
     this.keepRendering = true;
     this.model = new Model($('.rates_preview'), $('.rates_preview canvas'));
 
-    var scale = d3.scale.linear().domain([900, 2100]);
-
-    this.rollScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.pitchScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.yawScale = scale.range([Math.PI * 2, -Math.PI * 2]);
+    this.$superExpo = $('.pid_tuning input[name="show_superexpo_rates"]');
```

Comment:
@antstorm: Sorry for not getting back to you on this any sooner. I want to move this forward too, as you say there is a lot of potential for clean up. I think in order to move it forward, the first step has to be the establishment of a set of coding standards, and selection of one or more frameworks to use, and making these choices publicly known. I am open for suggestions and discussion on both counts.

Re the non - Betaflight support, I agree with you that we should not be supporting non - Betaflight. Re removing support for non - Betaflight, I am not sure how easy this is per se, as for older versions there is no distinction between Betaflight and cleanflight. What we should do is to decide on a sunset period (like 6 or 12 months), and remove support for versions that are older than this. Supporting users with not-the-latest versions is important, so that they can at least use the configurator to update their firmware and config, but this should have limits.